subset/cloud/test_udmi

```diff
@@ -23,8 +23,9 @@ device_id=`jq -r .device_id /config/device/module_config.json`
 
 # Do basic network connectivity check
 cat /etc/resolv.conf
-ping -c 2 172.217.164.106
-ping -c 2 pubsub.googleapis.com
+ping -c 2 172.217.164.106 || true
+ping -c 2 pubsub.googleapis.com || true
+(curl pubsub.googleapis.com 2>&1 > /dev/null  2>&1 && echo "Can reach pubsub.googleapis.com") || echo "Can't reach pubsub.googleapis.com" && false
```

Comment:
Doesn't this return false in all cases?  Either way, maybe better to add some ()'s or whever the moral equivalent is here just to be clear what the logic is.